### ISSUE TITLE
Import properties and property groups

### ISSUE THREAD

**Issue by Media-Evil** [mwganson/DynamicData#84]
Current implementation doesn't allow to import/transfer individual names or property groups from other FreecadObjects, including DD objects.

I found that import/transfer feature could be of use when I wrote some properties to a Group object (to keep the tree view clean), only to discover that if those properties are referenced from objects contained in that group, it causes cycle reference error. I had to re-create tens or properties (with tooltips) anew in a DD object.

**Comment by mwganson** [comment]
True.  We can import aliases from a spreadsheet or named constraints from a sketch, but not dynamic properties from another object at this time.  I will consider adding a feature to import dynamic properties from another object.  Note that already properties can be copied, but only one at a time with the copy/set command.
